src/compiler/checker.ts

```diff
@@ -3935,12 +3935,12 @@ namespace ts {
             return typeCopy;
         }
 
-        function forEachSymbolTableInScope<T>(enclosingDeclaration: Node | undefined, callback: (symbolTable: SymbolTable) => T): T {
+        function forEachSymbolTableInScope<T>(enclosingDeclaration: Node | undefined, callback: (symbolTable: SymbolTable, ignoreQualification?: boolean, scopeNode?: Node) => T): T {

```

Comment:
I dont see anywhere we are passing `ignoreQualification` true or false ? 
Also can we make these parameter nonOptional but type | undefined instead to ensure every call back considers what needs to be passed?

Reply:
Inside _this_ function, it's not; however the function we often pass as a callback uses it as a second parameter (and it's optional there) for its' _other_ callsites, so this optional parameter is here so the callback parameter is compatible with both that usage and the node-finding usage.